### PR TITLE
Clean up markup

### PR DESCRIPTION
**Pull request by johnholdun**
Mostly removes some presentational markup that is better-suited for CSS, like bullets between links, with an eye toward more flexible CSS in the future. There are no visual changes in this PR.

Also includes some little Handlebars convenience edits. And I did a pass on markup whitespace because I couldn't help myself.

### CODE REVIEW

File: src/pages/admin/followers.hbs [hunk 1]
```diff
@@ -7,11 +7,27 @@
   a per-bookmark setting on the bookmark edit page.
 </p>
 <form action="/admin/permissions" method="post">
-  <label> Auto-allow comments from (one username per line)</label><br />
-  <textarea name="allowed">{{allowed}}</textarea><br /><br />
-  <label> Auto-ignore comments from (one username per line)</label><br />
-  <textarea name="blocked">{{blocked}}</textarea>
-  <button type="submit">Save</button>
+  <div class="form-group">
+    <label for="permissions_allowed">
```
Comment (johnholdun) [inline src/pages/admin/followers.hbs:11]: Every label now has a `for` that corresponds to the `id` of the related input

File: src/pages/bookmark.hbs [hunk 2]
```diff
@@ -1,10 +1,8 @@
 {{> show_bookmark bookmark=bookmark }}
 
 <h2>Comments</h2>
-{{#if comments }}
```
Comment (johnholdun) [inline src/pages/bookmark.hbs:4]: The `each` block accepts an optional `else` block for exactly this use case

File: src/pages/partials/show_bookmark.hbs [hunk 3]
```diff
@@ -1,48 +1,64 @@
...
+        <span class="tag">
+          <a href="/tagged/{{this}}">
+            #{{this}}
+          </a>
+        </span>
       {{/each}}
     {{/if}}
   </div>
   <div class="bottom">
-    {{#if bookmark.comment_count includeZero=true}}
```
Comment (johnholdun) [inline src/pages/partials/show_bookmark.hbs:33]: I'm pretty sure that with `includeZero`, this `if` will _never_ return false, so I removed it, but correct me if I'm wrong!

Comment (ckolderup) [reply src/pages/partials/show_bookmark.hbs:33]: this is why I pay you the big bucks (big buck hunter arcade cabinets delivered to your home monthly)

**Comment by ckolderup** [comment]
thanks! rebase this off of main to pull in that bookmarklet fix and I think you should be good to go here?

**Comment by johnholdun** [comment]
Rebased!

**Review by ckolderup — APPROVED** [review]

**Pull request merged**